README.md

```diff
@@ -37,10 +37,9 @@ Here are some commands you can run:
 
 | Command | Description |
 | --- | --- |
-| yarn amo | Start the dev server/proxy (for amo) using data from Docker. _This is not currently working. See [issue-7196][]_. |
+| yarn amo:olympia | Start the dev server/proxy (for amo) using data from a local addons-server environment. |
```

Comment:
Do you think it's worth adding a note that the site will be located at `http://olympia.test:4000`? I know that is output to the console, so maybe not necessary, but it could be helpful for folks expecting it to be the same location as with `amo:dev` and `amo:stage`.

Reply:
Hmm, is it `http://olympia.test:4000` or `http://olympia.test:3333`? Both seem to work.

Reply:
`:3333` is the app, which runs behind the proxy at `:4000`.

Reply:
I think the output is fine. We don't develop with both local envs in parallel usually so I think it's for power users who would read the console output :)